parlai/core/worlds.py

```diff
@@ -481,7 +481,7 @@ def __init__(self, opt: Opt, agents, shared=None):
             # Add passed in agents directly.
             self.agents = agents
         self.acts = [None] * len(self.agents)
-        self.parley_mode = opt['parley_mode']
+        self.parley_mode = opt.get('parley_mode', 'round-robin')
```

Comment:
If this is in the flag default there is no need to add it here too. In fact, it might be a good check to see the code crash if the value is not set correctly.

Reply:
I actually added it here after the tests crash (it also crashed for all the other options I added). I'll take a closer to at the unit tests to see what's going on.